src/disco/topo/fd_topo.h

```diff
@@ -333,6 +333,9 @@ typedef struct fd_topo_t {
   fd_topo_link_t links[ FD_TOPO_MAX_LINKS ];
   fd_topo_tile_t tiles[ FD_TOPO_MAX_TILES ];
   fd_topo_obj_t  objs[ FD_TOPO_MAX_OBJS ];
+
+  ulong          agave_affinity_cnt;
+  ulong          agave_affinity_cpu_idx[ 1024 ];
```

Comment:
```suggestion
  ulong          agave_affinity_cpu_idx[ FD_TILE_MAX ];
```
And should they be ushort? Some places they are and some they aren't.